src/main/java/frc/robot/subsystems/claw/ClawSubsystemImpl.java

```diff
@@ -99,25 +105,25 @@ public void setWristPosition(double angle) {
     wristMotor.set(ControlMode.MotionMagic, angle * ClawConstants.DEG_TO_WRIST_POS);
   }
 
-  @Override
-  public boolean isConeMode() {
-    return isConeMode;
-  }
+  // @Override
+  // public boolean isConeMode() {
+    // return isConeMode;
+  // }
 
-  @Override
-  public void switchCargoMode() {
-    isConeMode = !isConeMode;
-  }
+  // @Override
+  // public void switchCargoMode() {
+  //   isConeMode = !isConeMode;
+  // }
 
-  @Override
-  public void setCargoModeCone() {
-    isConeMode = true;
-  }
+  // @Override
+  // public void setCargoModeCone() {
+  //   isConeMode = true;
+  // }
 
-  @Override
-  public void setCargoModeCube() {
-    isConeMode = false;
-  }
+  // @Override
+  // public void setCargoModeCube() {
+  //   isConeMode = false;
+  // }
```

Comment:
Do we need all this commented out code now that all this stuff is basically done in the NodeAndModeRegistry?

Reply:
no, but apparently team tradition is to comment out stuff "just in case we need it later"

Reply:
That is a tradition that should be abandoned - you can always find it in the revision history if you need it again later